docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang
+
+MAINTAINER Claudio D. Gasparini <claudio.gasparini@pantheon.sk>
+
+EXPOSE 179
+
+RUN apt-get update
+
+RUN go get github.com/osrg/gobgp/gobgp
```

Comment:
what about using the same version that is in glide?

Reply:
Done

Reply:
It depends on what is the purpose of this docker (quickstart, dev, rourte reflector only). Please move the files to subdirectory to express it and document it.